pom.xml

```diff
@@ -79,8 +79,8 @@
 			<plugin>
 				<artifactId>maven-compiler-plugin</artifactId>
 				<configuration>
-					<source>1.8</source>
```

Comment:
As far as I know Spring Boot 2.2.x still supports Java 8. Please don't force the library up to 11 - we may well do it eventually, but unless it is a requirement for Boot 2.2.x it doesn't belong in this issue or PR.